.github/workflows/test-sdk-update.yml

```
@@ -0,0 +1,49 @@
+name: Test SDK Update
+
+on:
+  repository_dispatch:
+    types: [sdk-update]
+
+jobs:
+  integration_test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: eu-central-1
+
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+
+      - name: Use Node.js 10
+        uses: actions/setup-node@v1
+        with:
+          node-version: 10
+
+      - name: Upgrade sdk to latest dev version
+        run: yarn install && yarn upgrade @kiltprotocol/sdk-js@dev
```

```suggestion
        run: yarn upgrade @kiltprotocol/sdk-js@dev
```
should be enough, install is implied. Requires that the lock file is up to date though.
If we want to pin the demo client to a specific sdk version, we would probably want to use 
```suggestion
        run: yarn add @kiltprotocol/sdk-js@dev
```
instead to make sure we overwrite the pinned version